test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -24,9 +24,14 @@ Describe "Get-Content" -Tags "CI" {
         Remove-Item -Path $testPath2 -Force
     }
 
-    It "Should throw an error on a directory  " {
+    It "Should throw an error on a directory" {
         { Get-Content . -ErrorAction Stop } |
-            Should -Throw -ErrorId "GetContentReaderUnauthorizedAccessError,Microsoft.PowerShell.Commands.GetContentCommand"
+            Should -Throw -ErrorId "GetContainerContentException,Microsoft.PowerShell.Commands.GetContentCommand"
+    }
+
+    It "Should throw error on a directory with specific message" {
+        { Get-Content . -ErrorAction Stop } |
+            Should -Throw "Unable to access '.' contents as it is a container. Please use 'Get-ChildItem .' instead."
```

Comment:
We never test a message because of localization. Please remove.

Reply:
Sure, will remove the tests, and it was not put in GetContentReadres() as we already have other check for verifying Tail and Header coexistence, So I thought to put it before GetContentReadres().